desk/lib/bolt.hoon

```diff
@@ -162,72 +160,56 @@
     =^  cards  yosh  (on-fail:ag term tang)
     [cards this]
   --
+  ::
   |%
-  ++  permissions
-    |_  [=bowl:gall state=state-0]
-    +*  parent  (sein:title our.bowl now.bowl src.bowl)
+  ++  is-allowed
+    |=  =bowl:gall
+    ^-  ?
+    ::
+    =*  parent  (sein:title our.bowl now.bowl src.bowl)
+    =*  user    src.bowl
+    =*  moon-gud  |(=(our.bowl parent) (~(has in whitelist) parent))
+    =*  moon-bad  |(=(our.bowl parent) (~(has in blacklist) parent))
+    ::
+    ::  Do not allow if in blacklist or kids and kid of blacklister
+    ?&  |(!(~(has in blacklist) user) &(kids !moon-bad))
```

Comment:
The way I read this, we're saying that *even if* the user is *explicitly* blacklisted, it should be let through under the condition that we allow kids and the moon's parent isn't blacklisted. To me this feels weird -- if someone is explicitly blacklisted, I want them out. But I haven't thought deeply about this and could probably be convinced. What's your reasoning for writing it the way it is?